app/components/support_interface/audit_trail_component.rb

```diff
@@ -19,18 +21,29 @@ def audits
         audits = audits.where(auditable_type: params[:auditable_type])
       end
 
-      audits.includes(:user).order('id desc').page(params[:page] || 1).per(60)
+      audits.includes(:user).order('created_at desc').page(params[:page] || 1).per(60)
     end
 
     attr_reader :audited_thing
 
   private
 
-    def audits_for_provider
-      standard_audits.or(Audited::Audit.where(
-                           auditable_type: 'ProviderRelationshipPermissions',
-                           auditable_id: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),
-                         ))
+    def provider_audits
+      audited_thing.own_and_associated_audits.unscope(:order).or(
+        Audited::Audit.where(
+          auditable_type: 'ProviderRelationshipPermissions',
+          auditable_id: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),
```

Comment:
A bit confused by this line..

I would have expected it to be either
> auditable **_id**: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id)**.id**,

or

>` auditable: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),` (sans id)

Reply:
I copied this code from what was before. Weirdly, passing just `auditable` doesn't work very well.